layers/b2qt/recipes-qt/automotive/qtapplicationmanager/sc-config.yaml

```diff
@@ -6,7 +6,7 @@ formatType: am-configuration
 ---
 containers:
   selection:
-    - com.pelagicore.radio: "softwarecontainer"
```

Comment:
so what happens to the radio app?

Reply:
There was no radio app in neptune3. There is webradio app but it doesn't matter what app to run in container as long as the configuration is correctly setup for that app

Reply:
explain in the title what you fix; fix, do, change, make are not good examples of words to use

Reply:
moreover, changes can be split